pages/linux/x11vnc.md

```diff
@@ -0,0 +1,23 @@
+# x11vnc
+
+> A VNC server that will enable VNC on an existing display.
+
+- Launch a VNC server that allows multiple clients to connect:
+
+`x11vnc -shared`
+
+- Launch a VNC server that will not exist after a client disconnects:
+
+`x11vnc -forever -viewonly`
```

Comment:
I am trying to understand the meanings of "forever" and "viewonly". I believe viewonly is for the server to close after the client disconnects. But what is forever for ?